ca/certificate-authority.go

```diff
@@ -156,7 +121,15 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	pa := policy.NewPolicyAuthorityImpl()
+	dbMap, err := sa.NewDbMap(paConfig.DBConnect)
+	if err != nil {
+		return nil, err
+	}
+
+	pa, err := policy.NewPolicyAuthorityImpl(dbMap, paConfig.EnforcePolicyWhitelist)
```

Comment:
Ohhhh, he's talking about the NewPolicyAuthorityImpl

I'm sitting here wondering why we aren't passing a PA into this function.